app/Http/Controllers/Admin/ContactUSSettings.php

```diff
@@ -4,95 +4,55 @@
 
 use Illuminate\Http\Request;
 use App\Http\Controllers\Controller;
-use Mapper;
 use App\ContactUsSetting,App\ContactInfoPhone,App\ContactInfoWhatsapp;
+use App\Repositories\SettingsRepositories as Setting;
+use App\Repositories\PhoneSettingsInfoRepositories as Phone;
+use App\Repositories\WhatsappSettingsInfoRepositories as Whatsapp;
+use Illuminate\Support\Facades\Input;
+
+use Mapper;
 use Session;
 
 class ContactUSSettings extends Controller
 {
-    public function show()
-    {
-        $check_settings = ContactUsSetting::count();
-
-        if ($check_settings >0) {
-            $info = ContactUsSetting::with(['whatsapp','phone'])->first();
+    protected $setting;
+    protected $phone;
+    protected $whatsapp;
 
-            $mape = Mapper::map($info->long,$info->lat, ['eventBeforeLoad' => 'console.log("before load");']);
-        }
-
-        return view('admin.contact_us.map',compact('mape','info'));
+    public function __construct(Setting $setting,Phone $phone,Whatsapp $whatsapp)
+    {
+        $this->setting = $setting;
+        $this->phone = $phone;
+        $this->whatsapp = $whatsapp;
     }
 
 
+
     public function create()
     {
 
-        $info = ContactUsSetting::with(['whatsapp','phone'])->first();
+        $info = $this->setting->with(['whatsapp','phone'])->findByFirst(['*']);
```

Comment:
move these queries to repos, maybe create a function in the `repo ->findPhone()`